modules/virt-node-placement-rule-examples.adoc

```diff
@@ -152,14 +152,16 @@ spec:
 [id="hostpathprovisioner-object-node-placement-rules_{context}"]
 == HostPathProvisioner object node placement rule example
 
-You can edit the `HostPathProvisioner` object directly or by using the web console.
+You can edit the `HostPathProvisioner` object directly or by using the web console. 
 
 [WARNING]
 ====
-You must schedule the hostPath provisioner and the {VirtProductName} components on the same nodes. Otherwise, virtualization pods that use the hostPath provisioner cannot run. You cannot run virtual machines.
+You must schedule the hostpath provisioner and the {VirtProductName} components on the same nodes. Otherwise, virtualization pods that use the hostpath provisioner cannot run. You cannot run virtual machines.
 ====
 
-The `HostPathProvisioner` object supports the `nodeSelector`, `affinity`, and `tolerations` node placement rules.
+If after deploying a virtual machine (VM) with the hostpath provisioner (HPP) storage class you remove the hostpath provisioner pod from the same node by using the node selector, you must first revert the change, at least for that specific node, and wait for the pod to run before trying to delete the VM.
```

Comment:
This sentence might cause localization problems because it is long and contains an unmarked subordinate clause. I recommend that you try breaking it into two or more shorter sentences.